### DISCUSSION TITLE
How to use Browser API with Solid Start

### DISCUSSION THREAD

**Discussion by jcubic** [solidjs/solid-start#1472] [Q&A] [answered]
I want to see if I can use Solid Start with jQuery Terminal (my library) so I can add this to the documentation. The problem is that when I want to use the standard code that I use everywhere I get an error that `window` is not defined.

```javascript
import $ from 'jquery';
import terminal from 'jquery.terminal';
import 'jquery.terminal/css/jquery.terminal.min.css';

terminal(window, $);
```

I use the starter (`npm init solid@latest`) and I've picked Solid Start.

I added the above code into app.tsx. 

So the question is how to use JavaScript DOM libraries with Solid Start? This code probably runs on the server since Solid Start use SSR, so how can I mark the code or a file as a client side?

**Comment by atk** [comment, marked as answer]
Use `!isServer` to only run your code on the client.

**Reply by jcubic** [reply]
Can this be run in top level of a file? or only in components?

**Reply by jcubic** [reply]
Also can you share the docs? I only found [clientOnly](https://start.solidjs.com/api/clientOnly)

When you google isServer it shows this page [api/useServerEvent](https://start.solidjs.com/api/useServerEvent) that is empty.

**Reply by atk** [reply]
https://docs.solidjs.com/reference/rendering/is-server

**Reply by jcubic** [reply]
I've seen this page, but where does it come from? Do I need to declare it as undefined const?

**Reply by jcubic** [reply]
It looks like the documentation is missing it's here:

https://docs.solidjs.com/reference/rendering/dev

**Reply by jcubic** [reply]
I've created a PR https://github.com/solidjs/solid-docs-next/pull/713
